### PR TITLE
90%: working multiple queues

### PR DESCRIPTION
**Pull request by omarqureshi**
Gets resque-web to work with multiple failure queues when using the 1.25.2 API with a fix around multiple failure queue backends (https://github.com/resque/resque/pull/1234)

This isn't tested with the resque 2.0 API, especially around the retrying of all jobs due to requeue_and_remove not accepting a queue name parameter.

Currently the Gemfile specifies the talis fork of resque, however, once the multiple failure queue backend fix has been merged in, this should be good to use with mainline resque.

### CODE REVIEW

File: app/controllers/resque_web/failures_controller.rb [hunk 1]
```diff
@@ -17,36 +18,43 @@ def destroy
...
 
     private
 
+    def requeue_queue(queue)
+      (Resque::Failure.count(queue)-1).downto(0).each { |id| requeue_single_job(id, queue) }
+    end
+
     #API agnostic for Resque 2 with duck typing on requeue_and_remove
-    def reque_single_job(id)
+    def requeue_single_job(id, queue)
```
Comment (camillef) [inline app/controllers/resque_web/failures_controller.rb:50]: +1 for correcting spelling

Comment (omarqureshi) [inline app/controllers/resque_web/failures_controller.rb:50]: DEATH TO AMERICAN SPELLING!

Comment (rsinger) [inline app/controllers/resque_web/failures_controller.rb:50]: Why do you think `que` is American?

Comment (omarqureshi) [inline app/controllers/resque_web/failures_controller.rb:50]: I stand corrected - Spanish then!

**Pull request merged**